Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -562,22 +538,47 @@ class EditProfileViewController: UIViewController {
         }
     }
     
+    private func setupTextFields() {
+        [nameTextField, pronounsTextField, hometownTextField, classTextField, phoneNumTextField, snackTextField, songTextField, stopTextField].forEach { textField in
+            textField.textField.delegate = self
+            textField.hidesLabel(isHidden: false)
+        }
+    }
+    
+    private func setupImagePicker() {
+        imagePicker.delegate = self
+        imagePicker.allowsEditing = true
+        imagePicker.mediaTypes = ["public.image"]
+        imagePicker.sourceType = .photoLibrary
+    }
+    
+    
+    
     // MARK: - Helper Functions
     
+    @objc private func updateProfileImage(_ sender: UITapGestureRecognizer) {
+        present(imagePicker, animated: true)
+    }
+    
+    private func convertImage (image: UIImage) -> String {
+        let base64 = image.jpegData(compressionQuality: 0.5)?.base64EncodedString() ?? ""
+        return "data:image/png;base64," + base64
+    }
```

Comment:
Can you extract this into a `UIImage+Extension` file? See below. I think that should work.

Reply:
```
func imageToB64(compression: CGFloat) -> String {
    let base64 = self.jpegData(compressionQuality: compression)?.base64EncodedString() ?? ""
    return "data:image/png;base64," + base64
}
```